keymap/tutorial.kbd

```diff
@@ -813,7 +813,7 @@
   -------------------------------------------------------------------------- |#
 
 (defalias
-  mt  (multi-tap 300 a 300 b 300 c 300 d e))
+  mt  (multi-tap 300 a 300 b 300 c 300 d e)
 
 
```

Comment:
That is the corresponding closing bracket of the one before `defalias`. This is a valid config and is usually written like that by lispers. If it really trips you, you can move that closing bracket to the next line.